dir.props

```diff
@@ -78,8 +78,8 @@
   <PropertyGroup>
     <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
       https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
-      https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json;
-      https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
+      https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json;
+      https://dotnetmygetlegacy.blob.core.windows.net/dotnet-core/index.json;
```

Comment:
why do we need the blob? I would expect both the `myget-legacy` feed and the blob to contain the same assets, right?

Reply:
> why do we need the blob? I would expect both the myget-legacy feed and the blob to contain the same assets, right?

The dotnet-core feed is a very large feed, so I opted to move it to sleet instead for ease of setup. The only packages in the dotnetmygetlegacy feed are those from dotnet-core. 

I would be open to moving some of those packages into the myget-legacy feed and eliminating this line in corefx, if you could identify the packages and version.